packages/react-component-library/src/components/CheckboxE/CheckboxE.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { Story, Meta } from '@storybook/react/types-6-0'
+import { Story, Meta } from '@storybook/react'
```

Comment:
I've just noticed the name of this set of component seems to be incorrect in the Sidebar.

Could we change this to Checkbox (Experimental).